StyleCop.Analyzers/StyleCop.Analyzers.Test/StyleCop.Analyzers.Test.csproj

```diff
@@ -14,6 +17,7 @@
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
     <TargetFrameworkProfile />
+    <NuGetPackageImportStamp>b08d3586</NuGetPackageImportStamp>
```

Comment:
:bulb: When you see these lines appear in the project file, it's safe to manually remove them *if* you are comfortable editing project files by hand. Don't worry about this one; I can remove it when I merge this pull request.